src/sfc/parser.js

```diff
@@ -98,13 +98,13 @@ export function parseComponent (
   function padContent (block: SFCBlock, pad: true | "line" | "space") {
     if (pad === 'space') {
       return content.slice(0, block.start).replace(replaceRE, ' ')
-    } else {
```

Comment:
I don't really think it's a good idea to remove the else condition here, since it's much much clearer to read & understand.